ws-nextjs-app/pages/[service]/live/[id]/Post/index.test.tsx

```diff
@@ -135,7 +136,8 @@ describe('Post', () => {
       ).toBeTruthy();
     });
 
-    it('should render the media player in a post containing video', async () => {
+    it('should render the legacy media player in a post containing video for the live environement', async () => {
+      process.env.SIMORGH_APP_ENV = 'live';
```

Comment:
So can I just confirm here - we're ok to use process.env in tests because these are running server side? But we now need to use getEnvConfig() for any logic that runs clientside?

Reply:
The `getEnvConfig` utility will work on both server and client side, but its only for reading the values rather than setting them.

Setting the `process.env` can be done like this on the server. The `getEnvConfig` has been setup to fallback to reading in the `process.env` if they aren't available on the `window.SIMORGH_ENV_VARS` object on the client.

Reply:
Ah cool, thanks for confirming!